astropy/units/format/latex.py

```diff
@@ -19,6 +19,10 @@ class Latex(base.Base):
     <https://www.iau.org/static/publications/stylemanual1989.pdf>`_.
     """
 
+    format_spec = ".8g"
```

Comment:
I intend to make the default values of these variables customizable.
Since ".8g" is too long for the axis label, especially with label text.
(May be ".1e" will be suitable for this use. e.g. `x (10^-8 A)`)
Is it OK to load these values from `astropy.config.ConfigNamespace`?

Reply:
Making them configuration parameters seems reasonable. Alternatively, a [``ScienceState``](https://docs.astropy.org/en/latest/api/astropy.utils.state.ScienceState.html?highlight=sciencestate).